x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/components/precision_slider/precision_slider.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiPanel,
+  EuiRange,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { RelevanceTuningLogic } from '../../relevance_tuning_logic';
+
+import { getStepDescription } from './utils';
+
+export const PrecisionSlider: React.FC = () => {
+  const {
+    searchSettings: { precision },
+  } = useValues(RelevanceTuningLogic);
+
+  const { updatePrecision } = useActions(RelevanceTuningLogic);
+
+  const stepDescription = getStepDescription(precision);
+
+  return (
+    <>
+      <EuiTitle size="m">
+        <h2>
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.title',
+            {
+              defaultMessage: 'Precision tuning',
+            }
+          )}
+        </h2>
+      </EuiTitle>
+      <EuiSpacer />
+      <EuiPanel color="subdued" hasShadow={false}>
+        <EuiText color="subdued">
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.description',
+            {
+              defaultMessage: 'Fine tune the precision vs. recall settings on your engine.',
+            }
+          )}{' '}
+          <EuiLink>
```

Comment:
Can we either add a documentation link here, or comment this out until we're ready to add a documentation link?

Reply:
@chriscressman @zumwalt do we know what the docs link/slug will look like for this?

@constancecchen I'll comment out for now

Reply:
I've commented this out in a84000b but also added a commented out test for this in b222f11

Reply:
@byronhulcher Can you include a screenshot that shows the context for the link to the doc?

It looks like this should be a link to general information about precision tuning, which we have here:

https://www.elastic.co/guide/en/app-search/current/precision-tuning.html (except replace `current` with specific docs version, such as `7.14`)

For 7.14, within the doc, we'll change:

> Tune precision per engine

To

> Tune precision per engine using Enterprise Search in Kibana
> ...
> Tune precision per engine using the search settings API